ratis-common/src/main/java/org/apache/ratis/RaftConfigKeys.java

```diff
@@ -61,6 +63,16 @@ static SupportedDataStreamType type(RaftProperties properties, Consumer<String>
     static void setType(RaftProperties properties, SupportedDataStreamType type) {
       set(properties::set, TYPE_KEY, type.name());
     }
+
+    String ENABLED_KEY = PREFIX + ".enabled";
+    boolean ENABLED_DEFAULT = true;
```

Comment:
If adding this config key is ok, what should be the default value? Is by default enabling the stream server ok?